### ISSUE TITLE
BN after first Conv2D

### ISSUE THREAD

**Issue by afzalxo** [scheckmedia/keras-shufflenet#6]
Hey! There is supposed to be a batch-norm between the first Conv2D and the 3x3 max-pool in stage1. The pre-trained weight file that I'm using [(ShuffleNetV1-1x-g=8.npz](http://models.tensorpack.com/ImageNetModels/ShuffleNetV1-1x-g=8.npz)) contains these parameters (gamma, beta, mean and variance for BN for conv1).
Could you please confirm if there is a BN in your pretrained model? I can't check since the link to your model is dead.

Thanks

**Comment by scheckmedia** [comment]
Hey, this implementation uses no batch norm for the stage 1, you can check the [model graph](https://raw.githubusercontent.com/scheckmedia/keras-shufflenet/master/model.png). You are right the link is dead but the weight file is located in the weight folder of the repository.

**Comment by afzalxo** [comment]
Got it... Thanks!
